lib/Settings/Settings.cpp

```diff
@@ -45,10 +49,10 @@ void Settings::onChange(const SettingType setting,
   listeners = new SettingListener(setting, callback, listeners);
 }
 
-void Settings::fireChange(const SettingType type) const {
```

Comment:
Instead of `setFor()` you should overload `fireChange()`.

Reply:
I'm not completely sure if this might bring us problems with implicit type conversions because of the [non-explicit constructor of std::bitset (2)](http://en.cppreference.com/w/cpp/utility/bitset/bitset).